src/integTest/groovy/com/github/jrubygradle/JRubyPrepareGemsIntegrationSpec.groovy

```diff
@@ -45,6 +47,28 @@ class JRubyPrepareGemsIntegrationSpec extends Specification {
             new File(jrpg.outputDir,"gems/slim-${SLIM_VERSION}").exists()
             new File(jrpg.outputDir,"specifications/.jrubydir").exists()
     }
+  
+    @Ignore // this test does not work with flatRepo since gems needs
```

Comment:
Are you commenting this out to execute ocally? I'm wondering if there's nothing we can do to get this test executed in Travis for each new PR